src/app/contextBroker/contextBroker.cpp

```diff
@@ -291,9 +291,9 @@ PaArgument paArgs[] =
   { "-httpTimeout",   &httpTimeout,  "HTTP_TIMEOUT",   PaLong,   PaOpt, -1,         -1,     MAX_L, HTTP_TMO_DESC      },
   { "-reqTimeout",    &reqTimeout,   "REQ_TIMEOUT",    PaLong,   PaOpt,  0,          0,     PaNL,  REQ_TMO_DESC       },
   { "-reqMutexPolicy",reqMutexPolicy,"MUTEX_POLICY",   PaString, PaOpt, _i "all",   PaNL,   PaNL,  MUTEX_POLICY_DESC  },
-  { "-writeConcern",  &writeConcern, "WRITE_CONCERN",  PaInt,    PaOpt, 1,          0,      1,     WRITE_CONCERN_DESC },
+  { "-writeConcern",  &writeConcern, "MONGO_WRITE_CONCERN",  PaInt,    PaOpt, 1,          0,      1,     WRITE_CONCERN_DESC },
 
-  { "-corsOrigin",       allowedOrigin,     "ALLOWED_ORIGIN",    PaString, PaOpt, _i "",          PaNL,  PaNL,     ALLOWED_ORIGIN_DESC    },
+  { "-corsOrigin",       allowedOrigin,     "CORS_ALLOWED_ORIGIN",    PaString, PaOpt, _i "",          PaNL,  PaNL,     ALLOWED_ORIGIN_DESC    },
   { "-corsMaxAge",       &maxAge,           "CORS_MAX_AGE",      PaInt,    PaOpt, 86400,          -1,    86400,    CORS_MAX_AGE_DESC      },
   { "-cprForwardLimit",  &cprForwardLimit,  "CPR_FORWARD_LIMIT", PaUInt,   PaOpt, 1000,           0,     UINT_MAX, CPR_FORWARD_LIMIT_DESC },
   { "-subCacheIval",     &subCacheInterval, "SUBCACHE_IVAL",     PaInt,    PaOpt, 60,             0,     3600,     SUB_CACHE_IVAL_DESC    },
```

Comment:
I didn't fix the indentation, so that you can see easily the changes.
Only OKed, I'll add a new commit aligning the columns (unless you don't want that, of course)

Reply:
Names seems to be fine, but I have one doubt...

What is the *efective* name of the env var? The one shown in this table plus the `ORION_` prefix (as I understand by `paConfig("prefix", (void*) "ORION_")`?

See the otuput of `contextBrokere -U` in this PR branch would help.

Reply:
The effective name is PREFIX + _env-var-name-in-array_, as you correctly understood.

E.g. "ORION_" + "ALLOWED_ORIGIN" == ORION_ALLOWED_ORIGIN

Reply:
Ok, perfect. NTC